sympy/stats/frv_types.py

```diff
@@ -66,17 +70,24 @@ def check(density):
         for p in density.values():
             _value_check((p >= 0, p <= 1),
                         "Probability at a point must be between 0 and 1.")
-        _value_check(Eq(sum(density.values()), 1), "Total Probability must be 1.")
+        val = sum(density.values())
+        _value_check(Eq(val, 1) != S.false, "Total Probability must be 1.")
```

Comment:
What's the benefit of using this than the original check?

Reply:
Same goes for the change here, its already in master